packages/clay-color-picker/src/Alpha.tsx

```diff
@@ -0,0 +1,111 @@
+/**
+ * SPDX-FileCopyrightText: © 2019 Liferay, Inc. <https://liferay.com>
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+import {ClayInput} from '@clayui/form';
+import ClaySlider from '@clayui/slider';
+import React, {useState} from 'react';
+
+type Props = {
+	/**
+	 * Callback function for when the hue value changes
+	 */
+	onChange: (event: any) => void;
+
+	/**
+	 * The inline styles to add on ClaySlider
+	 */
+	style: {
+		color: string;
+	};
```

Comment:
In that case since you just want `color` I would say just go with that property instead of having a property that only has the `style`, this is easier to use the component even though it is internal. We don't always need to follow the native attribute pattern to create React components, which makes things simpler, we can simplify some things.